python/hail/expr/types.py

```diff
@@ -555,12 +555,12 @@ def __init__(self, name, typ):
 
 class TStruct(Type):
     """
-    Hail type corresponding to :class:`hail.representation.Struct`.
+    Hail type corresponding to :class:`hail.utils.Struct`.
```

Comment:
I liked representation, although maybe the name is bad.  Objects which are runtime values and instances of types seems like a good and natural category of things.